http.go

```diff
@@ -65,9 +65,56 @@ func (k *Kasse) PostLoginPage(res http.ResponseWriter, req *http.Request) {
 	http.Redirect(res, req, redirect, http.StatusFound)
 }
 
+func (k *Kasse) GetDashboard(res http.ResponseWriter, req *http.Request) {
+	session, err := k.sessions.Get(req, "nnev-kasse")
+	if err != nil {
```

Comment:
Sind alle Fehler die hier auftreten können (und in session.Values[] darunter), Fehler, die von login behoben werden können?